ig/input/fsh/nbs-msh-segment.fsh

```diff
@@ -3,32 +3,32 @@ Id: segment-msh-logical-model
 Title: "MSH Segment"
 Description: "The MSH Segment"
 * segmentType 1..1 string "MSH"
-* fieldSeparator 1..1 string "MSH-1 The field separator for the message"
+* fieldSeparator 1..1 string "MSH-1 The Field Separator for the Message"
 * encodingCharacters 1..1 string "MSH-2 Encoding Characters"
-* sendingApplication 1..1 code "MSH-3 The sending application"
+* sendingApplication 1..1 code "MSH-3 The Sending Application"
 * sendingApplicationNamespaceId 1..1 code "MSH-3.1 Sending Application Namespace ID"
 * sendingApplicationUniversalId 1..1 string "MSH-3.2 Sending Application Universal ID"
 * sendingApplicationUniversalIdType 1..1 id "MSH-3.3 Sending Application Universal ID Type"
-* sendingFacility 1..1 code "MSH-4 The sending facility"
-* sendingFacilityNamespaceId 1..1 code "MSH-4.1 The sending facility namespace ID"
-* sendingFacilityUniversalId 1..1 string "MSH-4.2 The sending facility universal ID"
-* sendingFacilityUniversalIdType 1..1 id "MSH-4.3 The sending facility universal ID type"
-* receivingApplication 1..1 code "MSH-5 The receiving application"
-* receivingApplicationNamespaceId 1..1 code "MSH-5.1 The receiving application namespace ID"
-* receivingApplicationUniversalId 1..1 string "MSH-5.2 The receiving application universal ID type"
-* receivingApplicationUniversalIdType 1..1 id "MSH-5.3 The receiving application universal ID type"
-* receivingFacility 1..1 code "MSH-6 The receiving facility"
+* sendingFacility 1..1 code "MSH-4 The Sending Facility"
+* sendingFacilityNamespaceId 1..1 code "MSH-4.1 The Sending Facility Namespace ID"
+* sendingFacilityUniversalId 1..1 string "MSH-4.2 The Sending Facility Universal ID"
+* sendingFacilityUniversalIdType 1..1 id "MSH-4.3 The Sending Facility Universal ID Type"
+* receivingApplication 1..1 code "MSH-5 The Receiving Application"
+* receivingApplicationNamespaceId 1..1 code "MSH-5.1 The Receiving Application Namespace ID"
+* receivingApplicationUniversalId 1..1 string "MSH-5.2 The Receiving Application Universal ID Type"
+* receivingApplicationUniversalIdType 1..1 id "MSH-5.3 The Receiving Application Universal ID Type"
+* receivingFacility 1..1 code "MSH-6 The Receiving facility"
 * receivingFacilityNamespaceId 1..1 code "MSH-6.1 Receiving Facility Namespace ID"
 * receivingFacilityUniversalId 1..1 string "MSH-6.2 Receiving Facility Universal ID"
 * receivingFacilityUniversalIdType 1..1 id "MSH-6.3 Receiving Facility Universal ID Type"
-* messageDateTime 1..1 dateTime "MSH-7 The date and time the message was created"
-* messageType 1..1 Coding "MSH-9 The type of message"
+* messageDateTime 1..1 dateTime "MSH-7 The Date and Time the Message was Created"
+* messageType 1..1 Coding "MSH-9 The Type Of Message"
```

Comment:
Should the Of be uppercase here? It's not in the other segments below like line 31

Reply:
Thanks for catching this!